Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -578,244 +510,6 @@ public override bool ContainsBlock(HashDigest<SHA256> blockHash)
-            Guid chainId,
-            string key,
-            long lookupUntilBlockIndex)
-        {
-            if (_lastStateRefCaches.TryGetValue(
-                    chainId,
-                    out LruCache<string, Tuple<HashDigest<SHA256>, long>> stateRefCache)
-                && stateRefCache.TryGetValue(
-                    key,
-                    out Tuple<HashDigest<SHA256>, long> cache))
-            {
-                long cachedIndex = cache.Item2;
-
-                if (cachedIndex <= lookupUntilBlockIndex)
-                {
-                    return cache;
-                }
-            }
-
-            Tuple<HashDigest<SHA256>, long> stateRef =
-                IterateStateReferences(chainId, key, lookupUntilBlockIndex, null, limit: 1)
-                .FirstOrDefault();
-
-            if (stateRef is null)
-            {
-                return null;
-            }
-
-            if (!_lastStateRefCaches.ContainsKey(chainId))
-            {
-                _lastStateRefCaches[chainId] =
-                    new LruCache<string, Tuple<HashDigest<SHA256>, long>>();
-            }
-
-            stateRefCache = _lastStateRefCaches[chainId];
-
-            if (!stateRefCache.TryGetValue(key, out cache) || cache.Item2 < stateRef.Item2)
-            {
-                stateRefCache[key] = new Tuple<HashDigest<SHA256>, long>(
-                    stateRef.Item1,
-                    stateRef.Item2);
-            }
-
-            return stateRef;
-        }
-
-        /// <inheritdoc/>
-        public override IEnumerable<Tuple<HashDigest<SHA256>, long>> IterateStateReferences(
-            Guid chainId,
-            string key,
-            long? highestIndex,
-            long? lowestIndex,
-            int? limit)
-        {
-            highestIndex ??= long.MaxValue;
-            lowestIndex ??= 0;
-            limit ??= int.MaxValue;
-
-            if (highestIndex < lowestIndex)
-            {
-                var message =
-                    $"highestIndex({highestIndex}) must be greater than or equal to " +
-                    $"lowestIndex({lowestIndex})";
-                throw new ArgumentException(
-                    message,
-                    nameof(highestIndex));
-            }
-
-            byte[] keyBytes = RocksDBStoreBitConverter.GetBytes(key);
-            byte[] prefix = StateRefKeyPrefix.Concat(keyBytes).ToArray();
-
-            return IterateStateReferences(
-                chainId, prefix, highestIndex.Value, lowestIndex.Value, limit.Value);
-        }
-
-        /// <inheritdoc/>
-        public override void StoreStateReference(
-            Guid chainId,
-            IImmutableSet<string> keys,
-            HashDigest<SHA256> blockHash,
-            long blockIndex)
-        {
-            ColumnFamilyHandle cf = GetColumnFamily(_stateRefDb, chainId);
-            foreach (string key in keys)
-            {
-                byte[] keyBytes = StateRefKey(key, blockIndex);
```

Comment:
IMO We need delete `private byte[] StateRefKey(string stateKey, long blockIndex)` also.